README.md

```diff
@@ -10,25 +10,44 @@ WebdriverIO
 
 #### [Homepage](http://webdriver.io) | [Developer Guide](http://webdriver.io/guide.html) | [API Reference](http://webdriver.io/api.html) | [Contribute](http://webdriver.io/contribute.html)
 
-This library is a webdriver module for Node.js. It makes it possible to write
-super easy selenium tests in your favorite BDD/TDD test framework. It was
-originated by [Camilo Tapia's](https://github.com/camme) initial selenium project
-called WebdriverJS which was the first webdriver project on NPM.
+This library is a [webdriver](http://google-opensource.blogspot.com/2009/05/introducing-webdriver.html)
+(browser automation) module for Node.JS. It makes it possible to write
```

Comment:
One last thing: could we link to the W3 draft instead? https://w3c.github.io/webdriver/webdriver-spec.html